webapp/public/js/domjudge.js

```diff
@@ -93,266 +93,266 @@ var doReload = true;
 }
 
 function getScoreboard()
 {
-	var scoreboard = document.getElementsByClassName("scoreboard");
-	if (scoreboard === null || scoreboard[0] === null || scoreboard[0] === undefined) {
-		return null;
-	}
-	return scoreboard[0].rows;
+    var scoreboard = document.getElementsByClassName("scoreboard");
+    if (scoreboard === null || scoreboard[0] === null || scoreboard[0] === undefined) {
+        return null;
+    }
+    return scoreboard[0].rows;
 }
 
 function getRank(row)
 {
-	return row.getElementsByTagName("td")[0];
+    return row.getElementsByTagName("td")[0];
 }
 
 function getHeartCol(row) {
-	var tds = row.getElementsByTagName("td");
-	var td = null;
-	// search for td before the team name
-	for (var i = 1; i < 4; i++) {
-		if (tds[i].className == "scoretn") {
-			td = tds[i - 1];
-			break;
-		}
-	}
-	if (td === null) {
-		td = tds[1];
-	}
-	if (td !== null) {
-		if (td.children.length) {
-			return td.children[0];
-		}
-		return td;
-	}
-
-	return null;
+    var tds = row.getElementsByTagName("td");
+    var td = null;
+    // search for td before the team name
+    for (var i = 1; i < 4; i++) {
+        if (tds[i].className == "scoretn") {
+            td = tds[i - 1];
+            break;
+        }
+    }
+    if (td === null) {
+        td = tds[1];
+    }
+    if (td !== null) {
+        if (td.children.length) {
+            return td.children[0];
+        }
+        return td;
+    }
+
+    return null;
 }
 
 function getTeamname(row)
 {
-	var res = row.getAttribute("id");
-	if ( res === null ) return res;
-	return res.replace(/^team:/, '');
+    var res = row.getAttribute("id");
+    if ( res === null ) return res;
+    return res.replace(/^team:/, '');
 }
 
 function toggle(id, show)
 {
-	var scoreboard = getScoreboard();
-	if (scoreboard === null) return;
-
-	var favTeams = getSelectedTeams();
-	// count visible favourite teams (if filtered)
-	var visCnt = 0;
-	for (var i = 0; i < favTeams.length; i++) {
-		for (var j = 0; j < scoreboard.length; j++) {
-			var scoreTeamname = getTeamname(scoreboard[j]);
-			if (scoreTeamname === null) {
-				continue;
-			}
-			if (scoreTeamname === favTeams[i]) {
-				visCnt++;
-				break;
-			}
-		}
-	}
-	var teamname = getTeamname(scoreboard[id + visCnt]);
-	if (show) {
-		favTeams[favTeams.length] = teamname;
-	} else {
-		// copy all other teams
-		var newFavTeams = new Array();
-		for (var i = 0; i < favTeams.length; i++) {
-			if (favTeams[i] !== teamname) {
-				newFavTeams[newFavTeams.length] = favTeams[i];
-			}
-		}
-		favTeams = newFavTeams;
-	}
-
-	var cookieVal = JSON.stringify(favTeams);
-	setCookie("domjudge_teamselection", cookieVal);
-
-
-	$('.loading-indicator').addClass('ajax-loader');
-	$.ajax({
-		url: scoreboardUrl,
-		cache: false
-	}).done(function(data, status, jqXHR) {
-		processAjaxResponse(jqXHR, data);
-		$('.loading-indicator').removeClass('ajax-loader');
-	});
+    var scoreboard = getScoreboard();
+    if (scoreboard === null) return;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/mvr320/domjudge/pullRequest?prid=8427924)
```suggestion
	if (scoreboard === null) {return;}
```